crates/ruff_python_formatter/src/module/mod.rs

```diff
@@ -1,38 +1,36 @@
+use crate::context::PyFormatContext;
+use crate::{AsFormat, IntoFormat, PyFormatter};
+use ruff_formatter::Format;
+use ruff_formatter::{FormatOwnedWithRule, FormatRefWithRule, FormatResult, FormatRule};
```

Comment:
Nit: You can use `ruff_python_formatter::prelude::*` to get all relevant imports

```suggestion
use ruff_python_formatter::prelude::*;
```

Reply:
i'm a fan of having only explicit imports, it makes it easier to track what's from where and nowadays pycharm generates the right imports for me anyway